index.html

```diff
@@ -17,12 +17,14 @@
       <span id="seconds">00</span>
     </div>
     <div id="buttons-container">
-      <button>Start</button>
-      <button>Pause</button>
-      <button>Stop</button>
+      <button type="button" class="btn btn-start">Start</button>
+      <button type="button" class="btn btn-pause">Pause</button>
+      <button type="button" class="btn btn-atop">Stop</button>
```

Comment:
These classes are partially right, but the `btn-<string>` part is actually very specific to bootstrap. Bootstrap only supplies us with styling for: `btn-primary` `btn-secondary``btn-success``btn-danger` `btn-warning` `btn-info` `btn-light` and `btn-dark`